src/quests/prologue/tasks/autumnaton.ts

```diff
@@ -0,0 +1,13 @@
+import { Task } from "grimoire-kolmafia";
+import { $location, AutumnAton } from "libram";
```

Comment:
```suggestion
import { $locations, AutumnAton } from "libram";
```
to go with the rec below